app/src/main/java/com/aptoide/uploader/account/view/AutoLoginPresenter.java

```diff
@@ -125,4 +134,7 @@ private void handleOtherLoginsClick() {
         .doOnError(throwable -> uploaderAnalytics.sendLoginEvent("auto-login", "fail"))
         .andThen(Observable.empty());
   }
+  private boolean isInternetError(Throwable throwable) {
```

Comment:
internetError seems a bit broad and unusual choice of a name in my opinion. isNoNetworkError or isMobileNetworkError, my suggestions